Графика для всех страниц

Alephanta opened the pull request:
---
:mortar_board: [Проверяем рабочий процесс](https://up.htmlacademy.ru/htmlcss-individual/1/user/2402227/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/2402227-sedona-1/1/

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2402227/tasks/5)

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2402227/tasks/4)

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2402227/tasks/6)

Alephanta committed:
Add teacher

captain-zsa commented on Readme.md:
```diff
@@ -1,7 +1,7 @@
 
 * Студент: [Alexandra Kuznetsova](https://up.htmlacademy.ru/htmlcss-individual/1/user/2402227).
-* Наставник: `Неизвестно`.
+* Наставник: [Сергей Зубов] (https://htmlacademy.ru/profile/serj)
```
Тут нужно убрать пробел между ] и ( иначе работать не будет правильно.
<img width="647" alt="Снимок экрана 2023-09-20 в 09 24 07" src="https://github.com/htmlacademy-htmlcss/2402227-sedona-1/assets/18168845/e9bde370-aa09-4cb6-9ead-9bf59bf46d09">
вот тут это. Видишь, твоё норм отображается, а моё нет.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+        <ul class="navigation-list">
+          <li class="navigation-item">Главная</li>
```
Это всё ссылки внутри li. Главная без href, а остальные с href. Гостиницы с href="catalog.html" а О Седоне с href="#"

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+        </ul>
 
+        <ul class="navigation-user">
+          <li class="navigation-item">Поиск</li>
```
И это тоже ссылки.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+    </header>
+
+    <main class="main-page">
+      <h1 class="visually-hidden">Седона</h1>
```
А где это? 
<img width="470" alt="Снимок экрана 2023-09-20 в 09 27 41" src="https://github.com/htmlacademy-htmlcss/2402227-sedona-1/assets/18168845/ae6e7d9a-2d1a-4fb7-856e-bb705418094f">

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+      <h1 class="visually-hidden">Седона</h1>
+
+      <section class="description">
+        <h2 class="visually-hidden">Описание</h2>
```
Тут не описание, а список преимуществ. Ну или даже причин скажем так. Как раз тот заголовок и небольшой текст что выше на скриншоте показал – это и должно быть тут.

Alephanta replied:
done. тут благодраю, я поняла, поправила! сразу этого не увидела.

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+    <footer class="page-footer">
+      <section class="catalog-search">
+        <h2 class="page-footer-title">Поиск гостиницы в Седоне</h2>
+        <ul>
```
Не понятно список чего тут. Нет тут списка)) Элементов у списка тут нету никаких.

Alephanta replied:
вроде уже убрала - не могу найти , что не так 
список только у соц сетей сейчас в футере

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+
+    <footer class="page-footer">
+      <section class="catalog-search">
+        <h2 class="page-footer-title">Поиск гостиницы в Седоне</h2>
```
Ничего не понимаю... это же кнопка внизу текстов. Почему у тебя это заголовок? Заголовок в данной секции это Заинтересовались? и после него текст. Никаких тут не будет списков, ничего такого. Будет только вот заголовок, текст с описанием и кнопка. Всё.

Alephanta replied:
done поняла, я не сразу увидела по макету, что это кнопка

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+      </section>
+
+      <section class="newslatter">
+        <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
```
Давай добавим тут ещё и описание, которое есть в этом блоке

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,128 @@
+      <section class="footer-contacts">
+        <h2 class="visually-hidden">Контакты</h2>
+        <address class="footer-contacts-address">
+          <a class="footer-contacts-address-phone" href="tel:88128121212">+7 (812) 812-12-12</a>
```
Только не 8 в начале после tel: а tel+7

Alephanta replied:
done

captain-zsa reviewed (commented):
В целом хорошо, но есть некоторые нюансы, которые надо бы поправить)) Многих ошибок, которые делают студенты тут – у тебя нету) это радует.

Alephanta committed:
Sedona main page

Доработки
задание 1 - убрала пробелы
задание 2 - доработала Главную страницу

Alephanta committed:
Sedona main page

что сделано в коммите:
1 Только главная страница (каталог еще в процессе)
2 Добавлено все по комментариям
3 Выполнено задание раздела 2. Разметка
4 Выполнено задание раздела 3. Графика

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="">Главная</a>
```
Убрать href атрибут совсем.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <a class="navigation-logo" id="logo-icons" href="index.html">
```
Тоже убрать атрибут href. Мы уже на главной, а значит на неё снова переходить с неё же не нужно))

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="">
+              <span class="visually-hidden">Вход</span>
```
Всмысле вход? Какой вход? Куда? У нас отродясь такой кнопки не было в этом проекте))

captain-zsa replied:
Там совсем другая кнопка, с другим текстом и не скрытым.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="">
```
Интересно как это будет стилизоваться, если все классы у всех элементов меню одинаковы.

Alephanta replied:
со стилями совсем плохо - пока у меня нет ответа

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+
+      <section class="newslatter">
+        <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
+        <p>Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
```
А класс?))

Alephanta replied:
класс у тега p должен быть? вроде просто текст - получается всем тегам нужно класс присваивать, верно? 
тут об этом или какой-то другой класс ?

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+      <section class="newslatter">
+        <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
+        <p>Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
+        <form class="newsletter-form"> action="https://echo.htmlacademy.ru/" method="post">
```
Не замечаешь ошибки тут?)) Даже подсветка и в редакторе и в гитхабе показывает, что явно что-то не так.

Alephanta replied:
вроде нашла, поправила , а как посмотреть, что редактор ругается? я не вижу, в след раз вместе посмотрим.

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+            <span class="visually-hidden">e-mail</span>
+          </label>
+          <input class="field" placeholder="Ваш e-mail" name="newsletter-email" id="newsletter-email" required>
+            <button class="newslatter-button" type="submit">Подписаться</button>
```
Лишний таб

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+          <li class="footer-social-item">
+            <a class="button" href="https://vk.com/htmlacademy">
+              <span class="visually-hidden">Vkontakte</span>
+              <img class="footer-social-icons" id="social-icon-vkontakte" src="/images/social-icons/social-vk.svg" alt="Vkontakte" width="24,0005" height="14">
```
Декоративная картинка, не контентная.

Alephanta replied:
done поправила все картинки

captain-zsa commented on index.html:
```diff
@@ -5,8 +5,170 @@
+      <p class="site-production">
+        <a class="htmlacademy-link" href="https://htmlacademy.ru/">
+          <span class="visually-hidden">Сделано в</span>
+          <img class="footer-production-logo" id="logo-icons" src="/images/logo-icons/footter-logo.HTML Academy.svg" alt="Логотип HTML Academy" width="114,9" height="32,9">
```
Мы не указываем никогда десятые (и сотые тоже) доли пикселя. Не бывает их. Пиксель – это красный зеленый и синий светодиоды. Как подсветить половину этого пикселя? Включить красный, половину зеленого и не включать синий?))

Alephanta committed:
Задание Разметка

1. правки по комментариям
2. страница Каталог
3. добавлена графика
4. попытка стилизации, но лучше это не смотреть

Alephanta committed:
задание: 2.42 разметка для всех страниц_30.09.23

Добавила страницу Catalog
Поправила расположение некоторых тегов
Изменила названия некоторых тегов
Добавила тег article (ранее не было совсем)

Alephanta committed:
Изменения в блоках: Разметка + Графика

1 Скорректирована структура документа по секциям
2 Скорректированы теги - добавлен тег article
3 Добавлены картинки разного формата на главную и в каталог

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+          <a class="navigation-link">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="../catalog.html">Гостиницы</a>
```
Тут не нужен href, мы и так находимся на этой странице.

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+      <ul class="navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link">
+            <img class="navigation-search" src="../images/elements/icon-search.svg" alt="Форма поиска по сайту" width="18" height="18">
```
1. Это декоративное изображение. Не нужно его вставлять в разметку
2. Пути до картинок должны быть без ../ потому что папка images находится в той же папке что и файлы разметки

Alephanta replied:
Поняла, т.е. декоративные элементы через CSS добавляются - верно? 
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+      </a>
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link navigation-link-current">Главная</a>
```
Почему тут current? Мы ведь на странице Гостиницы. Тут должен быть href на index

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a class="navigation-logo">
```
Логотип является ссылкой на главную.

Alephanta replied:
done

captain-zsa replied:
Что done? Где href="index.html"?

Alephanta replied:
добавила

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+  <header class="page-header">
+    <nav class="navigation">
+      <a class="navigation-logo">
+        <img class="page-header-logo" id="logo-icons" src="../images/logo-icons/header-logo.svg" alt="Логотип города Седона" width="140" height="70">
```
Также как и везде – не должно быть тут ../

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link">
+            <img class="navigation-favourites" src="../images/elements/icon-favoriets.svg" alt="Избранные гостиницы" width="20" height="20">
```
Это также декоративное изображение.

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link">
```
Как так то? Разбирали ведь это даж на созвоне. Как нажать на кнопку которая внутри ссылки? Или на ссылку которая внутри кнопки? Статья есть же в академии.. не понимаю как так вышло.

Alephanta replied:
`<li class="navigation-item">
     <form action="/catalog.html" target="_blank">
            <button class="navigation-button" type="submit">Хочу сюда</button>
     </form>
 </li>`

Alephanta replied:
`<li class="navigation-item">
     <form action="/catalog.html" target="_blank">
            <button class="navigation-button" type="submit">Хочу сюда</button>
     </form>
 </li>`

Alephanta replied:
поправила, правда странно, что я не верно сделала 
не закрепилось

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link">
+            <button class="navigation-button" type="submit">Хочу сюда</button>
```
Почему type=submit ? У нас тут нет никаких форм. Отправлять ничего не надо))

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+
+  <main class="main-page">
+    <h1 class="visually-hidden">О Седоне</h1>
+    <img class="main-page-welcome-logo" src="../images/logo-icons/welcome-logo.svg" alt="Логотип городка Седона" width="458" height="352">
```
Не понял что это и для чего.

Alephanta replied:
эта часть для этого блока 
![image](https://github.com/htmlacademy-htmlcss/2402227-sedona-1/assets/52272164/0c78b1f3-cc9e-4623-b785-18f1cbda3ab9)

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+  </header>
+
+  <main class="main-page">
+    <h1 class="visually-hidden">О Седоне</h1>
```
У нас есть не скрытый заголовок над хлебными крошками. Он и должен быть тут.

Alephanta replied:
В каталоге поправила 
На гравной странице теги вроде правильные, но с картинками беда беда

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+
+            <!--какой-то сюрр с хлебными крошками-->
+
+            <ul class="breadcrumbs">
```
Табуляция съехала тут.

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+
+            <ul class="breadcrumbs">
+                <li class="breadcrumbs-item">
+                    <a class="breadcrumbs-home-link" href="../index.html">
```
Почему ../index ? catalog.html и index.html находятся на одном уровне (в одной и той же папке и лежат рядом друг с другом) не надо никаких выходов из папки делать для того чтоб добраться до index.

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+            <ul class="breadcrumbs">
+                <li class="breadcrumbs-item">
+                    <a class="breadcrumbs-home-link" href="../index.html">
+                        <img class="breadcrumbs-home" src="../images/breadcrumbs/home.svg" alt="Возврат на гравную" width="13" height="13">
```
Декоратив

captain-zsa replied:
Тут должна быть как раз подпись с visually-hidden Возврат на главную

Alephanta replied:
done, добавила visually-hidden тегу а

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+      <section class="catalog-hotels">
+            <h2 class="visually-hidden">Список гостиниц с фильтрами</h2>
+
+           <!--это тянет на логику JS не понимаю, как этот заголовок вывести <h2 class="hotel-list-total">Найдено гостиниц:</h2>-->
```
Никакой js логики мы не пишем. Мы то что видим – то размечаем и потом будем верстать, делаем так, как это сделано в макете и не думаем как и что там выводить и куда. Он есть в макете – раметили там где он должен быть. Логику мы писать сюда и не будем даже никакую.

Alephanta replied:
перенесла заголовок - найдено гостиниц - в другое место
 <ul class="hotel-list">
          <h2 class="hotel-list-total">Найдено гостиниц: 38</h2>
          <li class="hotel-card">

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+      </header>
+
+      <section class="catalog-hotels">
+            <h2 class="visually-hidden">Список гостиниц с фильтрами</h2>
```
Проблемы с табуляцией продолжаются. Табуляция должна быть 2 пробела, не больше. Везде проблемы по файлу. Посмотри поисправляй пожалуйста. Это стилистка кода. Обязательный критерий при сдаче проекта.

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                </fieldset>-->
+
+                <button class="button" type="submit">Применить</button>
+                <button class="button" type="submit">Сбросить</button>
```
Как так?)) мы отправим форму при нажатии на кнопку Сбросить. Так нельзя делать. Есть тип reset.

Alephanta replied:
done + добавила атрибут: value

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                <option value="expensive">Сначала дорогие</option>
+            </select>
+            <a class="button button-dark" href="#">
+                <span class="visually-hidden">Показать все карточки</span>
```
Не так. Отображать как карточки

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                <span class="visually-hidden">Показать все карточки</span>
+            </a>
+            <a class="button" href="#">
+                <span class="visually-hidden">Показать одну карточку</span>
```
Отображать как галерею

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                <span class="visually-hidden">Показать одну карточку</span>
+            </a>
+            <a class="button button-list" href="#">
+                <span class="visually-hidden">Показать список карточек</span>
```
Отображать как список

Alephanta replied:
done

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                <option value="cheap">Сначала дешевые</option>
+                <option value="expensive">Сначала дорогие</option>
+            </select>
+            <a class="button button-dark" href="#">
```
Почему dark? dark с английского – тёмный. Никак не про карточки))

Alephanta replied:
)) она просто темная самая, я ее дарк и назвала 
done поменяла на button-cards и еще добавила в след button-gallery (был пропущен класс)

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+            </a>
+            <ul class="hotel-list">
+                <li class="hotel-card">
+                    <a class="hotel-card-link" href="hotel1.html">
```
1. Мы не знаем страниц отелей, поэтому пишем href="#"
2. Именовать файлы и классы стоит без привязки к числам и всему подобному. Почему – потому что смотря на название такое не будет непонятно ровно ничего. Что за отель на картинке/странице? Пока не открыть картинку и не посмотреть – не поймешь. Более того режим сортировки поменяется и всё, уже этот отель будет не первым в списке и вся логика сломается. Именовать такого рода штуки нужно по названию того, что изображено на картинке или показано на странице.

Alephanta replied:
done 
спасибо, поняла!

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+            <ul class="hotel-list">
+                <li class="hotel-card">
+                    <a class="hotel-card-link" href="hotel1.html">
+                        <img class="hotel-card-image" src="../images/hotels/hotel-1.png" alt="Гостиница Амара резорт и спа" width="300" height="212">
```
Естественно, весь спич выше относится к названию изображения в том числе.

И да, почему это png то? Тут прям 100% jpg

Alephanta replied:
done 
названия изменила 
картинки все в правильном формате, путь неверно прописала

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                        <img class="hotel-card-image" src="../images/hotels/hotel-1.png" alt="Гостиница Амара резорт и спа" width="300" height="212">
+                        <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+                    </a>
+                    <p>Гостиница</p>
```
Класс и сюда и для цены тоже.

Alephanta replied:
done 
Добавила 2 класса 
<p class="hotel-type">Гостиница</p>
<p class="price-per-night">от 4000 Р</p>

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                    <p>от 1500 Р</p>
+                    <a class="holet-card-button button" href="#">Подробнее</a>
+                    <a class="holet-card-button button" href="#">В избранное</a>
+                    <!--не понимаю, как звезды и рейтинг добавить (рейтинг похоже текст, но! он же расчетный - значит JS, позже сделаю <img class="stars" src="../images/stars/star1.svg" alt="Звезды, присвоенные гостинице на основе международного гостиничного рейтинга">-->
```
Так, у нас интенсив про html и css сейчас. Мы НИЧЕГО не делаем по js. Давай договоримся, ты пока забудешь за его существование и будешь именно заниматься вёрсткой, а не думать что там и как с этим js и всем подобным.

` <div class="stars stars-4"><span class="visually-hidden">4 звезды</span></div>` вот так делаются звёзды, а рейтинг просто текст с классом а-ля hotel-card-rating

captain-zsa replied:
Ну и да, подобные вещи никогда не рассчитываются на фронте. Никакой js никогда рейтинг не должен рассчитывать, это бэкенд))

Alephanta replied:
done 
благодарю за коммент, про звезды было совсем не очевидно как сделать

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+                </li>
+                <li class="pagination-item">
+                    <a class="pagination-next" href="#">
+                        <span class="visually-hidden">Следующая страница</span>
```
Тут же 3 точки должны быть

Alephanta replied:
done (если правильно поняла)
имеется ввиду , что надо прямо так в прямом смысле [ ... ] поставить? 
мне было не поянтно, как это сделать - я и написала след сраница

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+        <p class="description">Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="newsletter-email">
+            <span class="visually-hidden">e-mail</span>
```
Можно просто этот класс повесить на label и не делать тут span.

Alephanta replied:
done 
поправила и в каталоге и на главной

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+            <span class="visually-hidden">e-mail</span>
+          </label>
+          <input type="email" placeholder="Ваш e-mail" name="newsletter-email" id="newsletter-email" required>
+            <button class="newslatter-submit" type="submit">Подписаться</button>
```
Опять же неверная табуляция.

Alephanta replied:
done 
Интересно, а есть какой-то плагин проверять табуляцию, может подскажешь? 
я проверяла через валидатор, не подсвечивал ничего такого

captain-zsa replied:
В этом месте так и не исправлено. Есть всякие beautify и прочая шелупонь, но откровенно оно плохо работает. Надо учиться видеть эти ошибки и исправлять. Тут должно быть на 1 таб меньше. input и button находятся в одном теге form на одинаковом уровне.

Alephanta replied:
done
я ранее думала, что button входит в input - поэтому табы везде стояли.

captain-zsa replied:
А почему теперь то всё обернуто в label? Зачем так делать то? Ты теперь вообще переделала всю разметку. Присылала же верный вариант в телеге.
<img width="803" alt="Снимок экрана 2023-10-04 в 18 10 20" src="https://github.com/htmlacademy-htmlcss/2402227-sedona-1/assets/18168845/54623a81-4974-455e-b511-ccb394ab6a11">
вот ведь так должно быть.

Alephanta replied:
у меня так же как на скрине , я не оборачивала ничего в label 
![image](https://github.com/htmlacademy-htmlcss/2402227-sedona-1/assets/52272164/434cfc78-7cc8-447c-b428-aea90d05f0a4)

captain-zsa commented on catalog.html:
```diff
@@ -0,0 +1,250 @@
+          <li class="footer-social-item">
+            <a class="button" href="https://vk.com/htmlacademy">
+              <span class="visually-hidden">Vkontakte</span>
+              <img class="footer-social-icons" id="social-icon-vkontakte" src="../images/social-icons/social-vk.svg" alt="Vkontakte" width="24" height="14">
```
Декоратив (то есть через css делается всегда такое).

Alephanta replied:
done 
убрала у соц сетей тег img

captain-zsa reviewed (commented):
images/logo-icons/footter-logo.HTML Academy.svg


никаких пробелов в названии изображений и больших букв. Ну и в целом явно тут можно название сделать сильно короче.

captain-zsa reviewed (commented):
А зачем вырезать абсолютно одинаковые звездочки? зачем star1.svg star2.svg и тд? Они же идентичны полностью. Одна картинка и всё. Не надо их вырезать кучу. Мы так никогда не делаем.

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,175 @@
+    <header class="page-header">
+      <nav class="navigation">
+        <a class="navigation-logo">
+          <img class="page-header-logo" id="logo-icons" src="../images/logo-icons/header-logo.svg" alt="Логотип города Седона" width="140" height="70">
```
Тут те же ошибки что и в каталоге. Писать идентичные не вижу смысла.

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,175 @@
+          <ul class="servises-list">
+            <li class="servises-item-house">
+              <h3 class="servises-title">Жилье</h3>
+              <img class="servises-item" id="home" src="../images/advantages-icons/icon-home.svg" alt="Жилье" width="75" height="72">
```
Декоратив. Значит через css делается.

Alephanta replied:
done 
убрала тег img

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,175 @@
+            </li>
+            <li class="servises-item-food">
+              <h3 class="servises-title">Еда</h3>
+              <img class="servises-item" id="food" src="../images/advantages-icons/icon-food.svg" alt="Еда" width="75" height="72">
```
Аналогично. И следующая тоже.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,175 @@
+        <p class="description">Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="catalog-search" href="../catalog.html">
+          <span class="visually-hidden">Поиск гостиницы в Седоне</span>
+          <button class="search-hotel-catalog" type="submit">Поиск гостиницы в Седоне</button>
```
Опять кнопка внутри ссылки. Не надо так.

Alephanta replied:
done

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,175 @@
+        <h2 class="title">Заинтересовались?</h2>
+        <p class="description">Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="catalog-search" href="../catalog.html">
+          <span class="visually-hidden">Поиск гостиницы в Седоне</span>
```
Точно такой же текст в самой кнопке. Для чего его дублировать?

Alephanta replied:
изменила, но добавила через form - хочу что-бы был переход, но он, почему-то, переходит по нажатию кнопки, хотя путь указан верно         
<form class="visually-hidden catalog-search" href="/catalog.html">Поиск гостиницы в Седоне
          <button class="search-hotel-catalog" type="button">Поиск гостиницы в Седоне</button>
</form>

Alephanta committed:
Правки на ревью от 02.10.2023

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,162 @@
+      <section class="catalog-details">
+        <h2 class="title">Заинтересовались?</h2>
+        <p class="description">Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <form class="visually-hidden catalog-search" href="/catalog.html">Поиск гостиницы в Седоне
```
Что это и почему тут форма? Никогда мы так не переходим с одной страницы на другую. Для этого существуют ссылки. Просто тег ссылки с таким текстом и href в каталог. Придуман какой-то лютый костыль для того действия, которое умеет делать обычный тег))

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,162 @@
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label class="visually-hidden" for="newsletter-email">e-mail</label>
+          <input type="email" placeholder="Ваш e-mail" name="newsletter-email" id="newsletter-email" required>
+            <button class="newslatter-submit" type="submit">Подписаться</button>
```
Также неверная табуляция в этом месте.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(../fonts/ptsans-400.woff2) format("woff2");
```
url надо заворачивать в кавычки. Всегда везде. И в background в том числе.

Alephanta replied:
done

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+    src: url(../fonts/ptsans-700.woff2) format("woff2");
+}
+
+@font-face {
```
Для чего тут italic я не понял. Во-первых его нет в файлах шрифтов, во-вторых на сайте он не используется нигде. Убрать подключения italic все.

Alephanta replied:
done

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+}
+
+@font-face {
+    font-family: "PT Sans Narrow";
```
Это что и почему у этого шрифта такой же файл для подключения как и у обычного pt sans?

PT Sans Narrow ≠ PT Sans, это 2 разных семейства. Ты тут делаешь так: сначала подключаешь правильно PT Sans, потом добавляешь ещё одно семейство PT Sans Narrow которое будет использовать абсолютно тот же файл что и PT Sans. А значит шрифты будут идентичны. А значит какой резон подключать ещё раз то же самое и делать ещё одно семейство просто ради семейства?

Alephanta replied:
удалили narrow вообще из css, потмоу что ег онет в загруженных шрифтах и ссылку не на что поставить

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+/* logo */
+.main-page-welcome-logo {
+    background-image: url(../images/pictures/Sedona-main.jpg);
+    width: "1200";
```
Размерности никогда не писались в кавычках. Посмотри пожалуйста свойство width в mdn и какие у него могут быть значения. Точно не то, что тут написано.

Ну и да, раздел базовой стилизации на то и раздел базовой стилизации, что мы не должны тут никаких задавать размеров элементов. Не надо. Больше придется потом удалять лишнего хлама только (чем эти width и heigth являются в данной ситуации).

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+    font-size: 18px;
+    line-height: 21px;
+    color: #333;
+    background-color: white;
```
Не пишем названием цвета никогда. Пишем #fff или #ffffff Всегда так должно быть. Потому что это более оптимально и искать это намного удобнее по коду, чем абстрактное слово white. У тебя может быть класс со словом white и поди найди среди россыпи этих white где то что тебе надо.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+
+.advantages-list,
+.servises-list {
+    list-style: none;
```
Советую сделать класc

.clear-list {
list-style-type: none;
margin: 0;
padding: 0;
}
и применять его в разметке тем тегам которым он нужен (ну для ol и ul). Это проще, чем каждый раз прописывать одни и те же свойства в стили ещё и плодить кучу доп селекторов через запятую вот в этом месте как у тебя сейчас.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+    list-style: none;
+}
+
+.advantages-item-city,
```
Определенно должен быть просто общий класс на эти 2 элемента и всё. Уникальности должны описываться в отдельных классах, идентичности – в общих.

Alephanta replied:
done

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+    color: #FFF;
+}
+
+.advantages-item-squere,
```
Абсолютно такой же принцип как и выше. У тебя вовсе получилось, что некоторые свойства пересекаются во всех классах. Очевидно надо сделать один общий, вынести туда те свойства, которые встречаются чаще всего, а в доп классах уже писать те свойства, которые более уникальны для каких-то мест.

Alephanta replied:
done

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+.advantages-item-devil {
+    background-color: #82B3D3;
+    padding: 103px 85px;
+    width: 400;
```
Опять же что тут за размеры и зачем они тут – не понятно и почему они так написаны – тоже непонятно. Не существует просто значения числом или тем более строкой в width и height.

Alephanta replied:
они могут быть в px или % - есть еще значения, но это основные 
я тут запуталась, потому что в уроках показывали в разметке можно добавить width / height и там нет px, они просто в кавычках

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,204 @@
+
+/* Разное */
+/* одна из попыток стилизовать блоки
+.advantages-list ul li:first-child {
```
Самые страшные селекторы которые можно использовать. Потому что вообще ничего не понятно, что тут стилизуется. Убрать это и забыть как страшный сон. Это невозможно использовать снова на любом другом элементе кроме вот этого огроменного набора из кучи вложенностей.

Alephanta replied:
ок, спасибо, поняла

captain-zsa reviewed (commented):
Не знаю, может прошлый раз не заметил. А почему есть welcome-logo.svg и welcome-sedona.svg ? Зачем нужны 2 картинки одинаковые почти полностью? Оставить нужно только полную картинку.

Дальше поехали по картинкам. Я говорил про спец символы и что их не надо использовать в названиях. Кавычка в images/pictures/Devil's-bridge.jpg очевидно не очень хорошо. И большие буквы тож.

Куча звездочек star1 star2  и тп так и остались почему-то. Оставить одну и всё. И не надо папку заводить на них отдельную. Там одна несчастная звездочка всего))

Alephanta committed:
Разметка, Графика_04.10.23

Поправила все комментарии.

Alephanta committed:
Разметка, графика_ 04.10.2023

Последняя правка к этому заданию, забыла

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,159 @@
+        </a>
+        <ul class="navigation-list clear-list">
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="#">Главная</a>
```
Тут не должно быть href.

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,159 @@
+            <a class="navigation-link navigation-link-current" href="#">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link">О Седоне</a>
```
А тут должен быть href=#

Alephanta replied:
href="# или путь на страницу" мы добавляем когда есть переход на страницу 
я добавила для главной, чтобы было подчеркивание страницы - без него подчеркивания нет

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,159 @@
+          <ul class="advantages-list clear-list">
+            <li class="item advantages-item-city">
+              <h3 class="advantages-title">Настоящий городок</h3>
+              <img class="description-item" id="tru-city" src="/images/pictures/tru-city.jpg" alt="Настоящий городок">
```
Зачем id в картинках? Я уже задавал этот вопрос.

Alephanta replied:
в разборе дз было про id для картинок
я не понимаю, когда все-таки надо добавлять id , а когда не надо

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,159 @@
+      <section calss="advantages">
+        <article class="advantages-list">
+          <ul class="advantages-list clear-list">
+            <li class="item advantages-item-city">
```
Слишком общее название item. Более конкретное надо. А то так недолго скатиться до того, что item будет и на странице каталога и в меню и тд и тп, потому что item переводится просто как пункт списка.

Alephanta replied:
Поменяла на items-list пока ничего больше в голову не приходит , но буду еще искать вариант

captain-zsa commented on index.html:
```diff
@@ -3,10 +3,159 @@
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label class="visually-hidden" for="newsletter-email">e-mail</label>
+          <input type="email" placeholder="Ваш e-mail" name="newsletter-email" id="newsletter-email" required>
+          <button class="newsletter-submit" type="submit">Подписаться</button>
```
Мдям... тут нормально всё. А почему в каталоге какая-то жесть?

Alephanta replied:
нашла, потому что в каталоге не исправила ( 
поправила

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+}
+
+.navigation-list {
+    font-family: "PT Sans", sans-serif;
```
Наследуется от body. Не надо его везде писать. Где он нужен ещё будет – это в button и input/textarea/select тегах. Там не наследуется. Хотя можно написать font-family: inherit; в конкретных селекторах и будет наследоваться тоже от родителя ближайшего где есть свойство (а ближайший будет body в нашем случае).

Alephanta replied:
отставила этот вариант - font-family: inherit;

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+
+.navigation-list {
+    font-family: "PT Sans", sans-serif;
+    color: inherit;
```
Бесполезно это делать если у тебя внутри меню ссылки. Нужно ссылкам задавать цвет текста, а не тому, что их оборачивает.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+/*Стилизация смысловых заголовков H3 уровня*/
+
+.advantages-title,
+.servises-title h2 {
```
А если таких мест будет ещё десяток? Не лучше ли тогда задать отдельный класс для таких элементов чтоб были общие свойства?

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+
+/*Стилизация смысловых описаний H3 уровня*/
+.advantags-description,
+.servises-description p {
```
Аналогично предыдущему.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+}
+
+/*Стилизация блоков Преимущества И Услуги*/
+.item {
```
То самое слишком общее название.

captain-zsa commented on styles/styles.css:
```diff
@@ -0,0 +1,184 @@
+.item {
+    padding: 103px 85px;
+    width: 400px;
+    height: 385px;
```
Убрать и width и height. Мы не занимаемся сеткой сейчас, не надо это пихать сюда. Более того height мы стараемся вообще не использовать или использовать минимально, потому что он рушит переполнение.

Alephanta replied:
done и поняла, почему у меня картинки и текст так странно смотрелись

Alephanta committed:
Исправления от 04.10.23 кроме id в картинках

Alephanta committed:
Разметка, графика, стилизация_05.10.2023_v1

1 В каталог добавлен раздел стоимости от -до
2 Добавлен фон - контентная картинка в раздел Подписка
